pkg/network/protocols/http2/protocol_testutils.go

```diff
@@ -0,0 +1,32 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux_bpf && test
+
+package http2
+
+import (
+	"unsafe"
+
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// GetHTTP2KernelTelemetry returns the HTTP2 kernel telemetry
+func (p *Protocol) GetHTTP2KernelTelemetry() (*HTTP2Telemetry, error) {
```

Comment:
Why do you think it is better over a utility test only?

Reply:
This is a good question.
So originally we added this as the tests using it was in the `tracer` package, and we didn't have access to USM's manager.
But now, as we are in the usm package, we are no longer limited, so I'll move it to a test utility indeed.